cli/providers/aws/logs/logs.go

```diff
@@ -0,0 +1,31 @@
+package logs
```

Comment:
should we call the package cloudwatch instead, to follow the pattern of the others? specially since we are only pulling log events from there.